City-of-Pori/pori-kada
# PORI-X: Attraction-list style fix

Attraction-list style fix, removed not used filter from listing view. Also compiled visitpori css, i think that was forgotten to do when removing the normalize from the theme.

To test:
1. drush fra -y && drush cc all
2. make sure you have attraction-card listings
3. go to one of those listing and confirm that the styles are working.
